packages/next/src/server/app-render/app-render.tsx

```diff
@@ -2535,7 +2527,6 @@ async function prerenderToStream(
             clientReferenceManifest.clientModules,
             {
               onError: serverComponentsErrorHandler,
-              nonce: ctx.nonce,
```

Comment:
these were probably added erroneously a while back and copied. nonce is not passed to RSC renderers because these renderers have no scripts they emit with implicit nonces unlike SSR where things like bootstrap scripts are nonced.